### PR TITLE
Fix brightness for 72x40 display

### PR DESCRIPTION
**Pull request by ew1abz**
72x40 OLED display uses a slightly different driver IC - SSD1306B. The display has only ~30% brightness if the internal Iref is turned off. To fix it, we need to issue a new command for this specific model.

### Useful info:
- https://bitbanksoftware.blogspot.com/2019/06/how-much-current-do-oled-displays-use.html?showComment=1572526557125#c3707851226502222177
- https://github.com/olikraus/u8g2/issues/1047

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,14 @@
...
 ## [Unreleased] - ReleaseDate
 
+### Fixed
+
+- Brightness for 72x40 display.
+
+### Added
+
+- SSD1306B datasheet.
+
```
Comment (jamwaffles) [inline CHANGELOG.md:16]: Please add a link to the PR here. I also don't think we need to show added docs in the changelog.

```suggestion
### Fixed

- [#141](https://github.com/jamwaffles/ssd1306/pull/141) Brightness for 72x40 display.
```

**Review by jamwaffles — CHANGES_REQUESTED** [review]
Sorry for the delay. Looks good to me overall, just a small tweak to the changelog needed.

I haven't tested this on a physical device, so I'll take your word that the new config works :grin:

**Comment by jamwaffles** [comment]
Hey, can you rebase this again please? There's another conflict in the changelog.

I missed that you had update the PR. When you're ready for a review, please hit the <kbd>Request review</kbd> button so I get a notification :).

I tested this on my half-dead 72x40 display and nothing seems broken, so I'll get this merged once the conflict is resolved.

**Comment by ew1abz** [comment]
No problem, James. I rebased and updated PR, ready for review.

**Review by jamwaffles — APPROVED** [review]
Looks good. Thanks!

**Comment by jamwaffles** [comment]
This is released in [0.4.2](https://crates.io/crates/ssd1306/0.4.2)

**Pull request merged**